EX-2033: Add timestamp in rpm version string

gaurangtapase opened the pull request.

gaurangtapase committed:
EX-2033: Add timestamp in rpm version string

KnightKu commented:
Shouldn't it be something likes following?
Version:	5.7.3.%{?build_timestamp}.ddn
Release:	8.g%{?rev}.%{?dist}

gaurangtapase commented:
yeah, we can do that as well, but I wanted to make sure we add the release before we add the timestamp in the version string.

gaurangtapase committed:
EX-2033: Add timestamp in collectd rpm version string

This patch modifies the format of the version string.
Also timestamp now is time since epoch.
